integration/manifests/executor.yaml

```diff
@@ -21,8 +21,12 @@ spec:
             - tail
             - -f
             - /dev/null
-          image: kubeapps/integration-tests:v1.0.4
+          image: kubeapps/integration-tests-pw:v1.0.0
           name: integration
           resources:
             requests:
-              memory: 100Mi
+              memory: 300Mi
+              cpu: 250m
+            limits:
+              memory: 600Mi
+              cpu: 500m
```

Comment:
Setting resource requests/limitations is never an exact science. Given that now in CircleCI we have access to a (slightly) better machine plus the random problems we were facing, I decided to increase the requested resources.
Ideally, we would need to check empirically which resources are used on every pod throughout the CI process, and give them a bit more room for the future, etc.

Reply:
Integration pod is executed inside the kind cluster, so it might "eat" resources from the Kubeapps installation if limits are not set. Probably it is an edge case, but I think it is a good practice to set some boundaries to resources consumption.
BTW I have seen this (new?) [feature of CircleCI](https://app.circleci.com/pipelines/github/kubeapps/kubeapps/5945/workflows/f409f42c-d658-4345-844b-a6b22ecd927b/jobs/73076/resources) that shows the resources usage of Docker.